docs/rest-api/resources/driveitem.md

```diff
@@ -24,6 +24,8 @@ For example:
 
 Items with the **folder** facet act as containers of items and therefore have a `children` reference pointing to a collection of **driveItems** under the folder.
 
+Items created under **lists** have a `listItem` reference pointing to the related **listItem** of **driveItem**. `drive/items/{item-id}/{listItem}` can be used to address the **listItem** resource.
```

Comment:
The wording used in the "Relationships" section for `listItem` is more accurate. 
The example - `GET /drive/items/{item-id}/children` - would be better located in the "Methods" section.

Reply:
Sorry, I don't think i understood you correct. Do you want me to remove that last sentence ?